src/serializer.ts

```diff
@@ -32,10 +32,31 @@ class Serializer {
     var object: any = transaction.toObject();
 
     // Convert fields names where direct conversion is possible.
-    this.convertPropertyName("account", "Account", object);
     this.convertPropertyName("sequence", "Sequence", object);
     this.convertPropertyName("signingPublicKeyHex", "SigningPubKey", object);
 
+    // Convert account field, handling X-Addresses if needed.
+    const account = transaction.getAccount();
+    if (account == undefined || !Utils.isValidAddress(account)) {
+      return undefined;
+    }
```

Comment:
Nitpicking here but I usually would write this as follows
```
if (!account || !Utils.isValidAddress(account)) return
```

Reply:
happy to convert to recommended style for the statement. 

I don't like omitting curly braces because I think it's error prone. Also assume you meant `return undefined` as opposed to `return`